operator/pom.xml

```diff
@@ -348,6 +348,11 @@
       <artifactId>jersey-test-framework-provider-inmemory</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>com.networknt</groupId>
+        <artifactId>json-schema-validator</artifactId>
+        <version>1.0.49</version>
```

Comment:
This new dependency should be okay. Apache license and earlier versions have been approved for other teams. Please move the version to a property in the top-level pom with the other version properties.

Reply:
Done.